.travis.yml

```diff
@@ -118,6 +118,10 @@ before_install:
     fi
 install:
   - if [[ "${TEST_SAGE}" != "true" ]]; then
```

Comment:
I misread this. It says "!=", so now this install pattern runs on every single test. I guess that maybe ok, but it'd be better if it only ran on a single test.